LayoutTests/fast/canvas/webgl/canvas-drawImage-expected.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="../../resources/js-test.js"></script>
+    <script src="resources/webgl-test.js"></script>
+    <script src="resources/webgl-test-utils.js"></script>
+    <script id="quadFragmentShader" type="x-shader/x-fragment">
+      precision highp float;
+      varying vec2 texCoord;
+
+      void main() {
+          gl_FragColor = vec4(texCoord, 0.5, 1.0);
+      }
+    </script>
+    <script>
+      "use strict";
+
+      var wtu = WebGLTestUtils;
+
+      function runTest(canvas)
+      {
+          const gl = wtu.create3DContext(canvas);
+          const program = wtu.setupProgram(gl,
+                                           [wtu.setupSimpleTextureVertexShader(gl), "quadFragmentShader"],
```

Comment:
No need to change this, but you probably didn't need a texture sample. The fragment shader could have been a ternary that draws red when gl_FragCoord.y < 0 and green otherwise.

Reply:
And then the expected.html wouldn't have even needed to use WebGL.